src/Illuminate/Validation/Concerns/ValidatesAttributes.php

```diff
@@ -356,9 +356,11 @@ public function validateBoolean($attribute, $value)
      * @param  mixed   $value
      * @return bool
      */
-    public function validateConfirmed($attribute, $value)
+    protected function validateConfirmed($attribute, $value, $parameters = [])
```

Comment:
Please add this param to the phpdoc too. :)

Reply:
I'va added it. Thank you very much for your advice 👍